apps/examples/wc-next10/next.config.js

```diff
@@ -30,6 +31,15 @@ const nextConfig = {
             ],
         });
 
+        //  "pie-icons-webc" bundling relies heavily on subpath exports to ship a  multi entry package.
+        //  next@10 and webpack 4 doesn't recognize this feature, resulting in Module not found errors
+        //  the following creates an alias to redirect imports to the correct path mimicking the native Subpath exports
+        //  supported in newer versions of nextjs
+        config.resolve.alias = {
+            ...config.resolve.alias,
+            '@justeattakeaway/pie-icons-webc': path.resolve(__dirname, '../../../node_modules/@justeattakeaway/pie-icons-webc/dist'),
```

Comment:
I'm not sure I 100% follow, but for any Next 10 applications outside the monorepo, would they have the same problems but not be able to use the same solution?

Reply:
they will face the same problem and this solution should work. I tested it in our next@10 here and CW

Reply:
CW next config already uses that for a while @xander-marjoram

Reply:
The packages in the example apps are symlink to the root packages if the versions match, I am not sure if that will make any difference